umake/frameworks/baseinstaller.py

```diff
@@ -114,6 +115,7 @@ def remove(self):
         if self.desktop_filename:
             with suppress(FileNotFoundError):
                 os.remove(get_launcher_path(self.desktop_filename))
+                os.remove(os.path.join(self.default_binary_link_path, self.desktop_filename.split('.')[0]))
```

Comment:
This is the removal of the link. We can create a get method, but other than that it works